src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,48 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added affinity/anti option to PODs with label dev.okteto.com/affinity
+- Add external resource icons
```

Comment:
Can we add a link to the docs?

Reply:
This needs to be merged to release-1.5 https://github.com/okteto/docs/pull/300 ?